addons/pkg-maven/jaxrs/src/main/java/org/commonjava/indy/pkg/maven/jaxrs/MavenContentAccessResource.java

```diff
@@ -163,4 +164,19 @@ public Response doGet(
         return handler.doGet( MAVEN_PKG_KEY, type, name, "", baseUri, request, new EventMetadata() );
     }
 
+    @Override
+    @ApiOperation( "Batch delete Maven artifact content under the given artifact store (type/name) and paths." )
+    @ApiResponse( code=200, message = "Batch delete operation finished." )
+    @ApiImplicitParam( name = "body", paramType = "body",
+                    value = "JSON array of paths list",
+                    required = true, dataType = "java.util.Set" )
+    @DELETE
+    @Path( "/clear" )
+    public Response doDelete(
```

Comment:
These seem functionally equivalent to the deletion of the store definition, other than leaving the empty repository in place. What's the use case for this?

Reply:
The case is to support to delete a batch of files from the repository. The API name "clear" maybe a litter bit confusing here :) , using "batchDelete" or just "batch" instead may work, WDYT ?

Reply:
I think this is for batch deletion where user pass a set of paths to be cleared from the target repo. Maybe we should rename it? "clear" sounds like to remove all. Maybe "/demote"?

Reply:
aha, you posted 1s before me.

Reply:
batchDelete or delete sounds good too.

Reply:
Can't we issue a DELETE to `/api/content/maven/hosted/some-repo/` (to the root path) and achieve the same thing as the /clear call though?